config/environment.js

```diff
@@ -2,7 +2,7 @@
 var fs = require('fs');
 function emberVersions() {
   return fs.readdirSync('ember').map(function(file) {
-    var matched = file.match(/^ember-(\d+\.\d+\.\d+)\.(\w+)/);
+    var matched = file.match(/^ember-(\d+\.\d+\.\d+)\.prod/);
```

Comment:
ah, i was actually started with this, but intended to match both `matched && matched[2] === 'prod'` guess i forgot that. But this works too!